### ISSUE TITLE
Write/Find Legal Disclaimer

### ISSUE THREAD

**Issue by nabeek** [nvalline/project-3#17]

**Comment by nabeek** [comment]
Jotted up something, but think we could discuss where to present the information. Maybe a link in the footer to a separate "Disclaimer" page?

```
The purpose of this application is to provide an easily-accesible, personalized snapshot of available information and resources on related to the COVID-19 pandemic.

Information collected during user creation is stored to present personalized metrics realted to COVID-19 and to ensure appropriate authorization to create, view, or delete events of potential SARS-CoV-2 exposure. Event data is not stored persistently. Once a user deletes an event all related information is deleted from the events database.

The resources and information provided is intended for educational purposes. If you suspect that you have been exposed to SARS-CoV-2 it is highly encouraged you seek medical attention.
```

**Comment by nabeek** [comment]
Added as an available component, closed via #87
